dockertest/output/texttable.py

```diff
@@ -18,7 +18,7 @@ class ColumnRanges(Mapping):
     """
 
     # Too few pub. methods, pylint doesn't count abstract __special_methods__
-    # pylint: disable=R0903
+    # pylint: disable=R0903, W0231
```

Comment:
W0231: 37,4: ColumnRanges.__init__: __init__ method from base class 'Mapping' is not called